selfswab/utils.py

```diff
@@ -59,3 +62,48 @@ def send_whatsapp_media_message(wa_id, media_type, media_id):
     )
     response.raise_for_status()
     return response
+
+
+def get_barcode_from_last_inbound_image(wa_id):
+    response = get_whatsapp_messages(wa_id)
+    media = None
+    for message in response["messages"]:
+        if (
+            message["type"] == "image"
+            and message["_vnd"]["v1"]["direction"] == "inbound"
+        ):
+            media = get_whatsapp_media(message["image"]["id"])
```

Comment:
Shouldn't we `break` after we get a media message?

Reply:
🤦